src/lib/1st/confetti/toggle-confetti.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import { tick } from 'svelte';
+
+  export let toggleOnce = false;
+  export let relative = true;
+
+  let active = true;
+
+  async function click() {
+    if (toggleOnce) {
+      active = !active;
+      return;
+    }
+
+    active = false;
+    await tick();
+    active = true;
+  }
+</script>
+
+<span on:click={click} class:relative>
```

Comment:
not sure if unintelligent but cannot find this button to toggle the confetti

Reply:
I want the whole screen to toggle it.